gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -158,6 +159,7 @@ enum
  * @brief IIO system paths
  */
 #define IIO_BASE_DIR "/sys/bus/iio/devices/"
+#define IIO_DATA_DIR "/dev/"
```

Comment:
This name may misleading readers because "datadir" usually means "/usr/share/...". You may want to use "IIO_DEV_DIR" or "DEV_DIR" instead.

Reply:
Done.